packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -51,7 +51,17 @@ const orderFormItemToOffer = (
   index?: number
 ): Indexed<IStoreOffer> => ({
   listPrice: item.listPrice / 100,
-  price: item.sellingPrice / 100,
+  price: item.priceDefinition?.calculatedSellingPrice
+    ? Math.floor(
+        (item.priceDefinition.sellingPrices.reduce(
+          (price, sellingPrice) =>
+            price + sellingPrice.quantity * sellingPrice.value,
+          0
+        ) /
+          item.quantity) *
+          100
+      ) / 10000
+    : item.sellingPrice,
```

Comment:
Preserve previous behavior

```suggestion
    : item.sellingPrice / 100,
```